e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js

```diff
@@ -441,6 +441,16 @@ describe('Keyboard Shortcuts', () => {
         cy.get('#moreDmModal').should('not.exist');
     });
 
+    it('MM-T4452 - CTRL/CMD+SHIFT+. Expand or collapse RHS', () => {
+        // # Type CTRL/CMD+SHIFT+. to open 'RHS'
+        cy.get('#post_textbox').cmdOrCtrlShortcut('{shift}.');
```

Comment:
As per the test steps written here https://automation-test-cases.vercel.app/test/MM-T4452

Scenario 1 : Executing shortcut on expanded RHS
- We first need to post a message
- Open the thread of the message by clicking on reply menu item
- Then execute the shortcut to expand the RHS
- Verify the RHS is fully expanded
- Execute shortcut again should bring RHS to non expanded position (but not close RHS)

Scenario 2 : Executing shortcut on closed RHS
- Execute the shortcut directly on closed RHS
- Verify un expanded RHS is now open
- Execute the shortcut again
- This time it should full expand RHS
- Executing it again should bring RHS back to non expanded position